### ISSUE TITLE
Table text appears blank

### ISSUE THREAD

**Issue by agrossfield** [JabRef/jabref#5854]
JabRef version 4.3.1 on Fedora 28

I have not reproduced this bug on the latest dev version, but the dev version is basically unuseable for me at this point, because features I need aren't there yet (e.g. pubmed/medline searching).  I was trying to use it, but I just can't at this point, and thus reverted back to the release.

Basically, when I open a library, the main list simply doesn't render -- it's as if there's nothing there. However, if I double click a line in the table, the entry editor opens just fine. I wondered if perhaps the font got changed to be white on white, or something weird like that, but when I try to open preferences, nothing happens.

Screen shot: 
![jabref](https://user-images.githubusercontent.com/13631218/72757233-6e05ce80-3b9d-11ea-9478-610239d78939.png)

**Comment by Siedlerchr** [comment]
You could try to reset the preferences using the command line switch. 
Please note that we all develop JabRef in our free time and therefore cannot support any older version. And as you note that it's not a problem in the latest development version, I will close this issue. 

However regarding the medline issue, could you please elaborate what is wrong, because there are several users who use this feature in the latest development version from https://builds.jabref.org and to my knowledge there isn't any issue.

**Comment by agrossfield** [comment]
As of a week or two ago, trying to use medline simply hung (and was documented as such).  However, I redownloaded this afternoon and that seems to work, so I can use the dev version.

Believe me, I appreciate that a) Jabref is a lot of work, and b) that you’re not paid or rewarded for it (I distribute open source code, so I know the pain of ungrateful users).  I’ve gotten hundreds of hours of help from Jabref over the ~15 years I’ve been using it.

Cheers,

Alan

________________________________
Dr. Alan Grossfield
Associate Professor
Department of Biochemistry and Biophysics
University of Rochester Medical Center
610 Elmwood Ave, Box 712
Rochester, NY 14642
Phone: 585 276 4193
http://membrane.urmc.rochester.edu
https://orcid.org/0000-0002-5877-2789


On Jan 20, 2020, at 4:13 PM, Christoph <notifications@github.com<mailto:notifications@github.com>> wrote:


You could try to reset the preferences using the command line switch.
Please note that we all develop JabRef in our free time and therefore cannot support any older version. And as you note that it's not a problem in the latest development version, I will close this issue.

However regarding the medline issue, could you please elaborate what is wrong, because there are several users who use this feature in the latest development version from https://builds.jabref.org<https://urldefense.proofpoint.com/v2/url?u=https-3A__builds.jabref.org&d=DwMCaQ&c=4sF48jRmVAe_CH-k9mXYXEGfSnM3bY53YSKuLUQRxhA&r=49qnaP-kgQR_zujl5kbj_PmvQeXyz1NAoiLoIzsc27zuRX32UDM2oX8NQCaAsZzH&m=ythX8nHPSdMxUFoeCMJIMsty0aZxNgI6kLEi3-fQsKE&s=U6PmDbNKeM3gFoqNKiCaB_YYWXiiQZaq8gcijrRO5eE&e=> and to my knowledge there isn't any issue.

**Comment by Siedlerchr** [comment]
Thanks for the kind words, we really appreciate long term users! 15 years of using JabRef is indeed a number! 
The medline fix didn't make it into the beta  version, however it was fixed a bit later.

**Comment by agrossfield** [comment]
The version I downloaded today seems much better from my perspective — things look very different, but nearly everything works.

Thank you for your efforts,

Dr. Alan Grossfield
Department of Biochemistry and Biophysics
University of Rochester Medical Center

On Jan 20, 2020, at 4:37 PM, Christoph <notifications@github.com> wrote:

﻿

Thanks for the kind words, we really appreciate long term users! 15 years of using JabRef is indeed a number!
The medline fix didn't make it into the beta version, however it was fixed a bit later.
